openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -26,10 +26,13 @@
 from edxmako.template import Template
 from microsite_configuration import microsite
 from xmodule.modulestore.django import modulestore
+from openedx.core.djangoapps.commerce.utils import ecommerce_api_client
 
 
 log = logging.getLogger(__name__)
 
+CREDIT_PROVIDERS_CACHE_KEY = 'credit.providers.api.data'
```

Comment:
I don't think it should be here as I guess we would need to show them on the dashboard as well.